book/website/collaboration/team-manual.md

```diff
@@ -0,0 +1,203 @@
+What are the [principles](https://www.lesleahlusko.org/principles-of-community-for-our-intellectual-family/)? 
+How do we do science in a way that fits with our mission/vision statement. 
+2. Roles and expectations: What are the responsbilities of each team member, including the PI? 
+When will these roles be discussed when a new member joins the team?
+
+### More details
+- Meta-processes: How is the lab manual updated? 
+Who can propose the changes (ideally every member) and what is the process? 
+Do you keep multiple versions of the lab manual? 
+Where will the lab manual be made available for all lab members to access?
+- Open Science: 
+How does the team manage and share code, data, materials? 
+How is documentation done? 
+- Research process: 
+how is research conducted at the lab? 
+How is the data analysed? 
+How is the code written, version controlled? 
+See for example the [Research toolkit by the Open Data Institute](https://theodi.org/article/check-out-our-rd-open-research-tools/). 
+Is work of lab members reproduced by other lab members (for example, to learn new techniques and to validate previous work)?
+- Communication: 
+How do people engage with each other? 
+- Under what condition can people provide their best work? 
+What support is needed? 
+How do people work? 
+Can they work remotely? 
+What are people's preferred working hours? 
+What to arrange when you go on a holiday? 
+Are there any weekly/daily tasks? 
+- Inclusivity: 
+How are hiring practices inclusive of minoritised groups? 
+- Resources: 
+anything that lab members need access to (servers, software, tools, room keys, library cards etc)
+- On/off boarding: 
+A list of what new members need to do when they start out, and a list what leaving members need to do before they go *(ref to on/off boarding)*. 
+- Behaviour: 
+Consider setting up a code of conduct or policy on scientific integrity (and align with the institutional/national policies on this - see below). 
+What should people do when they encounter harassment/discrimination? What is the lab culture? 
+- Credit: How to provide appropriate credit to the right people? 
+How to ensure that each contribution is recognised and rewarded? 
+This includes (lab) maintenance work, discussions, and ideas. 
+- Metrics: When is a lab member successful? 
+How do you discuss and measure this progress so that the lab member can more easily succeed? 
+How are successes celebrated? 
+How are reproducible, ethical and inclusive data science practices included in this assessment? 
+How is maintenance work evaluated and celebrated? 
+- How are mistakes handled (see [Rouder et al. 2019](https://doi.org/10.1177/25152459188019))? 
+How is insecurity being normalised? How is feedback given and acted upon?
+- Engagement: 
+what are the expectations around public outreach? 
+What is the social media policy for personal accounts? 
+How are all the materials made accessible (image descriptions, translations, etc (ref to chapter?) 
```

Comment:
etc is flagged by the latin checker - what do you usually turn this into? Also, this is a TODO to link the chapter you had in mind when writing this.

```suggestion
How are all the materials made accessible (image descriptions, translations, and the like (ref to chapter?) 
```